### ISSUE TITLE
What is the difference between 'Usage Policy' and License?

### ISSUE THREAD

**Issue by matgnt** [International-Data-Spaces-Association/ids-specification#75]
Since Usage Policies is the 'new' part of what IDS is capable of doing, I think it needs to be explained how it relates to 'licenses' like e.g. 'Creative Commons' and other, also proprietary licenses (paper contracts).

Opinions on that?

**Comment by PeterKoen-MSFT** [comment]
Licenses are a legal aspect and therefore part of the legal layer of a dataspace. However, a usage policy can reference a license or even enforce a license.
If a data plane is used where the enforcement of the license is technically not possible or insufficient then the obligation to follow the license agreement moves into the legal domain.
This is no different from data sharing agreements/licensing agreements without dataspaces. If one party violates the license there will be legal consequences.

**Comment by gbrost** [comment]
I totally agree: Usage policies are technical means to enforce or, if not possible, at least document usage agreements. They can refer to a license, but are not necessarily the same.
